src/App.jsx

```diff
@@ -14,45 +18,89 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+function getPreparedGoods(goods, sortType, isReverseD) {
+  const preparedGoods = [...goodsFromServer];
```

Comment:
You're creating a copy of 'goodsFromServer' instead of the 'goods' argument. This can lead to unexpected results because any changes to the 'goods' argument won't be applied to 'preparedGoods'. Replace 'goodsFromServer' with 'goods'.